oggm/utils.py

```diff
@@ -401,7 +401,7 @@ def gettempdir(dirname='', reset=False, home=False):
 
     basedir = (os.path.join(os.path.expanduser('~'), 'tmp') if home
                else tempfile.gettempdir())
-    return mkdir(os.path.join(basedir, 'OGGM', dirname), reset=reset)
+    return mkdir(os.path.join(basedir, dirname), reset=reset)
```

Comment:
Why changing this?

Reply:
Don't remember exactly why ... maybe due to an error I had. Or, probably it was to give more flexibility to the user to decide.